src/ert/config/queue_config.py

```diff
@@ -139,17 +113,130 @@ def generate_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
             )
 
 
-def _validate_torque_options(torque_options: List[Tuple[str, str]]) -> None:
-    for option_strings in torque_options:
+memory_options: Mapping[str, List[str]] = {
```

Comment:
Hmm, should we not categorize either by data format (string/float/int) or by topic, not by both?

Reply:
I see, memory has its own syntax. Should we not call this a string option, and then apply an extra layer of validation after the format of the value has been accepted?

Reply:
I suggest adding a map from options to regular expressions that are also verified in a loop, and include only memory_per_job as of now.

Reply:
For better code understanding I've decided to keep it explicit.